lib/public/views/About/About.js

```diff
@@ -11,19 +11,88 @@
  * or submit itself to any jurisdiction.
  */
 
-import { Observable } from '/js/src/index.js';
+import { getRemoteData } from '../../utilities/fetch/getRemoteData.js';
+import { Observable, RemoteData } from '/js/src/index.js';
 
 /**
- * Model representing handlers for homePage.js
+ * Model representing handlers for aboutPage.js
  */
-export default class Overview extends Observable {
+export class AboutModel extends Observable {
     /**
-     * The constructor of the Overview model object
-     * @param {Object} model Pass the model to access the defined functions
-     * @returns {Object} Constructs the Overview model
+     * The constructor for the About model object
+     * @param {Object} model Pass the parent model to access the defined functions
+     * @returns {Object} Constructs the About model
      */
     constructor(model) {
         super();
         this.model = model;
+        this._appInfo = RemoteData.NotAsked();
+        this._dbInfo = RemoteData.NotAsked();
+    }
+
+    /**
+     * Fetches information about the all services related to bookkeeping by querying the status api
+     * @returns {void}
+     */
+    async fetchAllServiceInfo() {
+        this._dbInfo = await this.fetchServiceInfo('/api/status/database');
+        this._appInfo = await this.fetchServiceInfo('/api/status/gui');
+        this.notify();
+    }
+
+    /**
+     * Fetches information about the service determined by the api path
+     * @param {string} apiPath the path to the endpoint for this service's status
+     * @returns {Object} an object which contains the version (if the request was successful)
+     *                   and the status of the service
+     */
+    async fetchServiceInfo(apiPath) {
+        let service = RemoteData.loading();
+        try {
+            const { data: info } = await getRemoteData(apiPath);
+            service = RemoteData.success({
+                version: info.version,
+                status: info.status,
+            });
+        } catch {
+            service = RemoteData.failure({ status: { ok: false } });
+        }
+        return service;
+    }
+
+    /**
+     * Set the app info and notify
+     * @param {RemoteData} appInfo information about the app
+     * @returns {void}
+     */
+    set appInfo(appInfo) {
```

Comment:
As the setter is not being used at all, I would recommend to remove it. 
There is no point in allowing modification if we do not plan to do it